### PR TITLE
FIX-TEMPLATE: 101-sig-image-version-create

### PR DESCRIPTION
**Pull request by javid-spektra**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: 101-sig-image-version-create/prereqs/prereq.azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+    },
+    "adminPassword": {
+      "type": "securestring"
+    }
+  },
+  "variables": {
+    "galleryName": "[uniqueString(resourceGroup().id)]",
+    "imageName": "[concat(variables('galleryName'), '/', parameters('galleryImageDefinitionName'))]",
+    "vnetId": "[resourceId('Microsoft.Network/virtualNetworks', parameters('virtualNetworkName'))]",
+    "subnetRef": "[concat(variables('vnetId'), '/subnets/', parameters('subnetName'))]"
```
Comment (bmoore-msft) [inline 101-sig-image-version-create/prereqs/prereq.azuredeploy.json:78]: simplify, just use resourceId()

File: 101-sig-image-version-create/prereqs/prereq.azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+            "max": "64"
+          },
+          "memory": {
+            "min": "2048",
+            "max": "307720"
+          }
+        }
+      }
+    },
+    {
```
Comment (bmoore-msft) [inline 101-sig-image-version-create/prereqs/prereq.azuredeploy.json:120]: remove all the unused resources (everything but the gallery and image)

File: 101-sig-image-version-create/README.md [hunk 3]
```diff
@@ -9,8 +9,8 @@
...
+[![Deploy To Azure US Gov](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg?sanitize=true)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-sig-image-version-create%2Fazuredeploy.json)  
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-sig-image-version-create%2Fazuredeploy.json)
 
+This template creates an Image Version for an Image Definition. Please ensure that you have deployed the 101 templates to create a Shared Image Gallery and the Image Definition before deploying this template.
```
Comment (bmoore-msft) [inline 101-sig-image-version-create/README.md:16]: update this section - I don't think you need to call out the samples but just what the prereqs do (or you can mention your prereq template)

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: 101-sig-image-version-create/prereqs/prereq.azuredeploy.json (hunk 2 above)
Comment (javid-spektra) [reply 101-sig-image-version-create/prereqs/prereq.azuredeploy.json:120]: What resources are not being used?

File: 101-sig-image-version-create/prereqs/prereq.azuredeploy.json (hunk 1 above)
Comment (akhilthomas011) [reply 101-sig-image-version-create/prereqs/prereq.azuredeploy.json]: Done

**Pull request merged**